frontend/src/scratch/Scratch.module.css

```diff
@@ -24,35 +24,37 @@
 
 .diff {
     display: block;
+    border-spacing: 1em 0;
```

Comment:
I wanted to add horizontal `border-spacing` to put some padding around the line number column. Setting the vertical spacing to `0` is actually less than we had before -- making the diff a bit denser. 

I like the denser diffs better, but I can imagine that the original line spacing was intentional? I'm fine reverting this, and putting the padding in `.diffLineNumber` instead.

Reply:
> I can imagine that the original line spacing was intentional?

It wasn't! I'm happy with a denser diff personally.